config/initializers/cycle_timetables.rb

```diff
@@ -37,12 +37,29 @@
     show_deadline_banner: Time.zone.local(2025, 7, 1, 9), # 12 weeks before Apply deadline
     apply_deadline: Time.zone.local(2025, 9, 16, 18),
     reject_by_default: Time.zone.local(2025, 9, 24, 23, 59, 59), # 1 week and a day after apply deadline
-    find_closes: Time.zone.local(2025, 10, 1, 23, 59, 59), # The evening before the find opens in the new cycle
+    find_closes: Time.zone.local(2025, 9, 30, 23, 59, 59), # The evening before the find opens in the new cycle
     holidays: {
       christmas: Date.new(2024, 12, 18)..Date.new(2025, 1, 5),
       easter: Date.new(2025, 3, 18)..Date.new(2025, 4, 1),
     },
   },
+  # The following dates are still TBD as of 12 August 2024
+  # - Show summer recruitment banner
+  # - Show deadline banner
+  # - Holidays
+  2026 => {
+    find_opens: Time.zone.local(2025, 10, 1, 9), # First Wednesday of October
```

Comment:
Why do we need to explicitly state that it's the first Wednesday of October?  Similar questions for the comments below.

I'd prefer it if we removed the comments from line 46 to line 49 and put TBC in line like we have on Find. I think that looks neater and keeps the codebases consistent. What do you think?

https://github.com/DFE-Digital/publish-teacher-training/blob/main/app/services/find/cycle_timetable.rb#L43